packages/stream_chat_flutter/lib/src/user_avatar.dart

```diff
@@ -79,6 +79,8 @@ class UserAvatar extends StatelessWidget {
                   imageUrl: user.extraData['image'] as String,
                   errorWidget: (_, __, ___) =>
                       streamChatTheme.defaultUserImage(context, user),
+                  placeholder: (_, __) =>
+                      streamChatTheme.placeholderUserImage!(context, user),
```

Comment:
pass null into placeholder if `placeholderUserImage` is null, Otherwise, it will throw due to `!`

Reply:
The tests are also failing due to this